pkg/controller/worker/machines_test.go

```diff
@@ -494,7 +494,7 @@ var _ = Describe("Machines", func() {
 						},
 					}
 				}
-				It("should return the expected machine deployments when disableExternal IP is true with profile image types", func() {
+				XIt("should return the expected machine deployments when disableExternal IP is true with profile image types", func() {
```

Comment:
XIt? :)

Reply:
Yes, there was an issue while mocking the ListGCPMachinesCall. I have fixed it and now all the existing tests are working and this line is no longer `XIt()` :)